app/templates/flatpages/default.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
+
+{% block head %}
+<title>{{ flatpage.title }} | HackItForward</title>
+{% endblock %}
+
+{% block body %}
+<div class="content" style="margin-top: 60px"> {{ flatpage.content }}</div>
```

Comment:
`px` aren't the best units for offsets like this because they're a fixed unit. Does this look good on mobile?

Reply:
well it was more of a temporary fix, I want to rework the UI later on.

Reply:
Use `<space>` or `<space class="large">`.

Reply:
<img width="1186" alt="Screen Shot 2020-10-01 at 6 45 36 PM" src="https://user-images.githubusercontent.com/43485866/94870719-4d716200-0416-11eb-8218-10216a2bdbfe.png">

Your proposed solution doesn't work as you can see above. I just tested it, the current code looks perfect on mobile.
<img width="496" alt="Screen Shot 2020-10-01 at 6 47 05 PM" src="https://user-images.githubusercontent.com/43485866/94870813-83164b00-0416-11eb-9374-d93017a1f25a.png">

Reply:
Oh, I didn't mean to put the text inside of a `<space>` I meant use a `<space>` instead of the inline style.

Reply:
Sorry, I don't understand what you mean. I have never used <space>. Is it a 1 line fix?

Reply:
Apparently it's a Cirrus thing, it might not be big enough tbh, but try this.
```suggestion
<space></space>
<div class="content">{{ flatpage.content }}</div>
```